proto/entities/v1alpha/entities.proto

```diff
@@ -16,6 +16,11 @@ service Entities {
 
   // Triggers a best-effort discovery of an entity.
   rpc DiscoverEntity(DiscoverEntityRequest) returns (DiscoverEntityResponse);
+
+  // Finds the list of local entities whose titles match the input string.
+  // A fuzzy search is performed among documents, groups and accounts. 
+  // For groups and documents, we match the title, while we match alias in accounts.
+  rpc SearchLocalEntites(SearchLocalEntitesRequest) returns (SearchLocalEntitesResponse);
```

Comment:
I’d omit the “Local” prefix in the RPC name, as most of these APIs are local.

Reply:
Sure, Just wanted to clear the ambiguity between this call and `DiscoverEntity`